### PR TITLE
Fix regression branch

### PR DESCRIPTION
**Pull request by WallyMaier**
## Proposed Changes
A recent change from #1417, which I merged in had the regression set to feature_cat_wall  in the TestCases. This will update and fix that call.  I'm sure I will need to check additional fixes associated with this change.

### CODE REVIEW

**Review by pcarruscag — CHANGES_REQUESTED** [review]
Ok wait. @bigfooted also merged something to testcases with new restarts for the work in axisymmetry fix.

**Comment by WallyMaier** [comment]
@pcarruscag @bigfooted I can see the changes. It looks like a restart filename changed.  But I can hold off on fixing the regression value and merging the changes.

File: .github/workflows/regression.yml [hunk 1]
```diff
@@ -128,7 +128,7 @@ jobs:
         uses: docker://ghcr.io/su2code/su2/test-su2:220614-1237
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c feature_cat_wall -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c develop_tmp -s ${{matrix.testscript}}
```
Comment (pcarruscag) [inline .github/workflows/regression.yml:131]: Update what you need to update but do not change this branch to develop for now

**Review by pcarruscag — APPROVED** [review]

**Comment by WallyMaier** [comment]
@pcarruscag I saw this got merged.  What do you think the next steps are with develop_tmp?

**Comment by pcarruscag** [comment]
We remove it after #1784 is merged

**Pull request merged**